packages/flutter/test/material/modal_bottom_sheet_test.dart

```diff
@@ -251,6 +251,40 @@ void main() {
     semantics.dispose();
   });
 
+  testWidgets('Verify that visual properties are passed through', (WidgetTester tester) async {
```

Comment:
Should also provide similar tests: specifying the theme's bottomSheetTheme, explicitly creating a BottomSheet with parameters overrides the theme, and a regression test that verifies the configuration of a BottomSheet's Material (we want to ensure that the appearance of a default BottomSheet doesn't change).

Reply:
Added tests for all of this in `bottom_sheet_theme_test.dart`